pkg/osinfo/osinfo_darwin.go

```diff
@@ -0,0 +1,35 @@
+package osinfo
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+func GetOSInfo() (OSInfo, error) {
```

Comment:
In all the OSInfo variations, you already know the OS type. Suggest to remove the error and always return some info

Reply:
Good point. Done.